lib/client/conntest/database/database.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package database
+
+import "github.com/gravitational/trace"
+
+// PingRequest contains the required fields necessary to test a Database Connection.
+type PingRequest struct {
```

Comment:
Request prefix indicates tat this is some kind of massage content but is is just a `PingRequest` just contains a configuration for dialing database